src/core_modules/capture-core/components/WidgetFeedback/WidgetFeedback.types.ts

```diff
@@ -0,0 +1,39 @@
+export type FilteredText = {
+    id: string;
+    message: string;
+    color?: string | undefined;
```

Comment:
Remove the "undefined" type because using "?" is sufficient. Change it everywhere in this file.